### PR TITLE
Add kubevirt-web-ui-components

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: frontend/public/components/factory/list-page.jsx [hunk 1]
```diff
@@ -162,7 +162,10 @@ export const FireMan_ = connect(null, {filterList: k8sActions.filterList})(
           </Link>;
         } else if (createProps.items) {
           createLink = <div className="co-m-primary-action">
-            <Dropdown buttonClassName="btn-primary" id="item-create" title={createButtonText} items={createProps.items} onChange={(name) => history.push(createProps.createLink(name))} />
```
Comment (rawagner) [inline frontend/public/components/factory/list-page.jsx]: add generateFunction(name) to kubevirt package which will either call history.push or functions

File: frontend/package.json [hunk 2]
```diff
@@ -63,12 +64,16 @@
...
     "js-base64": "^2.4.5",
     "js-yaml": "3.x",
+    "kubevirt-web-ui-components": "^0.1.1",
     "lodash-es": "4.x",
     "openshift-logos-icon": "1.7.0",
     "patternfly": "^3.45.0",
     "patternfly-react": "^2.16.0",
     "patternfly-react-extensions": "2.5.1",
     "plotly.js": "1.28.x",
+    "postcss-class-prefix": "^0.3.0",
```
Comment (mareklibra) [inline frontend/package.json]: please clean-up

File: frontend/public/components/factory/list-page.jsx (hunk 1 above)
Comment (mareklibra) [reply frontend/public/components/factory/list-page.jsx]: and localize change to a separate PR, please

File: frontend/public/kubevirt/components/vm.jsx [hunk 3]
```diff
@@ -4,10 +4,15 @@ import React, { Component, Fragment } from 'react';
 import { ListHeader, ColHead, List, ListPage, ResourceRow, DetailsPage } from './factory/okdfactory';
```
Comment (mareklibra) [inline frontend/public/kubevirt/components/vm.jsx]: Separate PR, please

File: frontend/public/kubevirt/components/vm.jsx [hunk 4]
```diff
@@ -243,13 +248,71 @@ export const VirtualMachinesDetailsPage = props => <DetailsPage
   ]}
 />;
 
-export class VirtualMachinesPage extends Component {
+const mapStateToProps = ({k8s}) => ({
+  k8s: k8s
```
Comment (mareklibra) [inline frontend/public/kubevirt/components/vm.jsx]: shorter `k8s` is enough here

**Comment by rawagner** [comment]
split to #16 #17 and #18. Closing

**Pull request closed without merging**